docs/examples/vlm_pipeline_api_model.py

```diff
@@ -130,6 +178,12 @@ def main():
     #     prompt="OCR the full page to markdown.",
     # )
 
+    # Example using the OlmOcr (dynamic prompt) model with Ollama:
+    # (uncomment the following lines)
+    # pipeline_options.vlm_options = ollama_olmocr_vlm_options(
+    #     model="hf.co/mradermacher/olmOCR-7B-0225-preview-GGUF:Q8_0",
```

Comment:
Let's please use official models in the example, i.e. `allenai/olmOCR-7B-0225-preview`